.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Commits polluted with project/compilation files (.idea, bin etc.).
It makes tracking actual changes more difficult.